<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FangControl</class>
 <widget class="QMainWindow" name="FangControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>503</width>
    <height>332</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FangControl</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0" rowspan="5">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="mode_button">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Mode Select</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_2">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="lineWidth">
         <number>5</number>
        </property>
        <property name="midLineWidth">
         <number>5</number>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="2" column="0">
          <widget class="QLabel" name="walk_g1_m">
           <property name="text">
            <string>Walk - Gait 1</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="rotate_m">
           <property name="text">
            <string>Stationary - Rotate</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="translate_m">
           <property name="text">
            <string>Stationary - Translate</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="1">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>168</width>
        <height>105</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="1">
     <widget class="QFrame" name="frame_3">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>10</verstretch>
       </sizepolicy>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="lineWidth">
       <number>5</number>
      </property>
      <property name="midLineWidth">
       <number>5</number>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="static_label">
         <property name="text">
          <string>Status:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="status_label">
         <property name="text">
          <string>Not Connected</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="connect_button">
      <property name="text">
       <string>Connect</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QGroupBox" name="keypad_groupBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="2" column="2">
        <widget class="QPushButton" name="center_button">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Center</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="up_button">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Forward</string>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QPushButton" name="stand_button">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Stand</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="l1_button">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>+Z</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="left_button">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Left</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="sit_button">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Sit</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="r1_button">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>-Z</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QPushButton" name="right_button">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Right</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QPushButton" name="down_button">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Backward</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="1">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>168</width>
        <height>50</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>503</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>mode_button</sender>
   <signal>clicked()</signal>
   <receiver>FangControl</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>173</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>200</x>
     <y>61</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_connect_button_clicked(bool)</slot>
 </slots>
</ui>
